dictionaries/de_DE/README.md

```diff
@@ -47,4 +47,7 @@ The Hunspell source for this dictionary can be found:
 
 ## License
 
-MIT
+GPL-3.0-or-later
+
+> Some packages may have other licenses included.
+> See [src/hunspell/license](./src/hunspell/license) and [src/German_de_DE.txt](./src/German_de_DE.txt)
```

Comment:
Links in the dictionary READMEs need to be a full URL, since they get published to NPM as is and will just link to a 404.

Like: https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/de_DE/src/German_de_DE.txt

Reply:
I had copied this pattern from another README, so I've also gone through and updated the other READMEs that were the same way